cime_config/testdefs/testmods_dirs/clm/clm60_monthly_matrixcn_soilCN30/user_nl_clm

```diff
@@ -1,2 +1,2 @@
 use_soil_matrixcn = .true.
-paramfile = '$DIN_LOC_ROOT/lnd/clm2/paramdata/ctsm60_params_cn30.c240822.nc'
+paramfile = '/glade/campaign/cgd/tss/people/oleson/modify_param/ctsm60_params_cn30.c241017.nc'
```

Comment:
- [ ] @slevis-lmwg will copy file and ./rimport it
- [ ] @olyson will update the path